docs/Parallel-Learning-Guide.rst

```diff
@@ -462,6 +462,17 @@ See `the lightgbm_ray documentation`_ for usage examples.
 
   ``lightgbm_ray`` is not maintained by LightGBM's maintainers. Bug reports or feature requests should be directed to https://github.com/ray-project/lightgbm_ray/issues.
 
+Mars
+^^^^
+
+`Mars`_ is a tensor-based framework for large-scale data computation. LightGBM integration, maintained within the Ray GitHub repository, can be used to perform distributed LightGBM training using ``pymars``.
```

Comment:
```suggestion
`Mars`_ is a tensor-based framework for large-scale data computation. LightGBM integration, maintained within the Mars GitHub repository, can be used to perform distributed LightGBM training using ``pymars``.
```

I think this was a copy-paste mistake from the `lightgbm_ray` docs.

Reply:
Ah, sorry! Indeed typo. Thanks for catching!